ConsoleProject/Application.cs

```diff
@@ -6,8 +6,8 @@ namespace ConsoleProject {
     class Application {
 
         [STAThread]
-        public static void Main() {
-            new Frontend().ShowForm();
+        public static void Main(string[] arguments) {
+            new Frontend().ShowForm(arguments);
```

Comment:
I don't like that we are passing "arguments" to the frontend.
Renaming them to "filelist" would be better in the ShowForm method signature.

However the ultimate goal is to seperate the console and the frontend layers. Thus the frontEnd should not require the console inputs. The underlying layers (eg texture converter) should be able to service both the windows form frontend and the console frontend.

Reply:
If this is only a naming issue, fine, it can be renamed.
However, I would prefer to keep the possibility to pass files as arguments, not necessarily via console input, but rather as a drag-and-drop on the exe or double-click with the exe set as the default opening program.

Reply:
"double-click with the exe set as the default opening program." 
Haven't thought about this, good point. 

At the Application layer we can work with it as arguments, thats fine.
But the form should already know what is it getting as parameter, thus the need of rename.

Some sort of validation might be a good idea too. Perhaps add a todo here, to validate the input arguments.